backend/src/server.js

```diff
@@ -1,57 +1,22 @@
 require('dotenv').config({ path: '.env' });
```

Comment:
I'm not sure shrinking the main server file down to nothing should be our goal here?
I feel by abstracting it into the serverConfig might not be the best way of doing this, plus it is more than a config, so serverConfig seems like bad semantics.

Reply:
i am about to make a mess in here, so it might be good.